src/libraries/Microsoft.PowerFx.Core/Public/Values/NamedValue.cs

```diff
@@ -25,7 +44,19 @@ public NamedValue(KeyValuePair<string, FormulaValue> pair)
         public NamedValue(string name, FormulaValue value)
         {
             Name = name ?? throw new ArgumentNullException(nameof(name));
-            Value = value;
+            _backingDType = value.Type._type;
```

Comment:
>value.Type._type [](http://example.com/codeflow?start=28&length=16)

The Value getter indicates that _value can be null. Will this crash here?

Reply:
you are correct, thanks for spotting it.